srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableEntry.java

```diff
@@ -61,13 +61,12 @@ public static VariableEntry withRetrievedValues(final Collection<? extends Varia
         	{
         		continue;
         	}
-            // TODO: Exchange the switch statement for a better solution.
+        	// TODO: Exchange the switch statement for a better solution.
             switch(v.getRetrievalKey())
-            {	
-//	            case 1:
-//	        		variableEntry.fDynamicValues.put(v.getKey(), patient.getGender()
-//        				.equalsIgnoreCase("M")? "male": "Female");
-//	        		break;
+            {
+	            case 1:
+	        		variableEntry.fDynamicValues.put(v.getKey(), patient.getGender());
```

Comment:
Mmmm these lines are tantalizingly similar to case 2 now, but I don't think we can improve it yet.